exec/Chainweb/Backfill.hs

```diff
@@ -33,7 +32,7 @@ backfill :: Env -> IO ()
 backfill e@(Env _ c _ _ cids) = withPool c $ \pool -> do
   cont <- newIORef 0
   mins <- minHeights cids pool
-  let !count = M.size mins
+  let count = M.size mins
```

Comment:
Banging nearly everything in a `let` is a practice I adopted from Niklas Hambuchen to avoid inlining GOTCHAs. It's probably not the end of the world here, though.

Reply:
In this case it's forced on the next line by the conditional.  I tend to think that sprinkling bangs everywhere unnecessarily obscures the times when they're actually important.